src/features/contactverzoek/ContactverzoekFormulier.vue

```diff
@@ -250,16 +298,16 @@ watch(
   ([el, hasContact]) => {
     if (!el) return;
     el.setCustomValidity(
-      hasContact ? "" : "Vul minimaal een telefoonnummer of een e-mailadres in",
```

Comment:
check even of je alle aanbevolen extensies hebt. het lijkt erop dat prettier niet gedraaid wordt (formatteer tool voor de frontend). daarna even alle gewijzigde frontend bestanden openen en opslaan. maakt het PR een stuk leesbaarder.

Reply:
ik heb ze beide installeert en die komma's moeten toch gewoon weg.

![image](https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/assets/132667147/81034ba9-d22b-4f24-8690-e95cf7c68093)

Reply:
de komma's moeten blijven staan.